pages/a/ms-azure-guest-post.html

```diff
@@ -0,0 +1,89 @@
+title: Crate guest post for Azure - Setting up Azure for Crate
+author: Chris Ward
+description: tbc
+created: 2016-02-04
+status: publish
+post_type: post
+tags: a, b
+category: developernews
+
+# Crate guest post for Azure - Setting up Azure for Crate
+
+Crate.io has been striving to create the most scalable database in our industry. We aim to make scaling your data from millions to billions of records and beyond, as simple as possible.
+
+We have clients who write and query gigabytes of data a minute, store terabytes a day and love the performance and stability that Crate offers. But the question always remained at the back of our minds, when taken to the extreme, what is Crate really capable of?
+
+Our aims were to:
```

Comment:
I'm not sure if these were the initial aims, but more aims we now have after the process. I'd love to know what our aims really were in the first place, in addition to just "wanting to know what Crate could do".